gensim/utils.py

```diff
@@ -1146,15 +1146,16 @@ def keep_vocab_item(word, count, min_count, trim_rule=None):
         else:
             return default_res
 
-def check_output(*popenargs, **kwargs):
+def check_output(*popenargs, stdout=subprocess.PIPE, **kwargs):
```

Comment:
I'm bit doubtful about a workaround for this, it gave a syntax error for python2 in above check.
If I specify `stdout=subprocess.PIPE` as first argument for making py2 compatible, it won't work for other wrappers as they have `cmd`(basic command) as their first argument